docs/source/documentation/tube-shape-derivative/tube-shape-derivative.rst

```diff
@@ -67,6 +67,7 @@ files are in the mesh directory.
   fout = File("output/u.pvd")
```

Comment:
Note that this file changes as a consequence of rebuilding the documentation. The git repository shouldn't really have generated files in it, but that's a different problem from this PR.